app/views/auth.py

```diff
@@ -0,0 +1,35 @@
+from flask import render_template, request, Blueprint, flash, redirect, url_for
+from flask_login import login_user, logout_user
+from werkzeug.security import check_password_hash
+
+import app.models.users as users_model
+
+
+bp = Blueprint("auth", __name__)
+
+
+@bp.route("/login", methods=["POST", "GET"])
+def login():
+    if request.method == "GET":
+        return render_template("login.html")
+    else:
```

Comment:
nit; no need to write `else` since the above `if` statement does have a `return`

Reply:
This has fixed